giskard/scanner/issues.py

```diff
@@ -25,6 +26,42 @@ class IssueGroup:
     description: str
 
 
+class ScanExamples(ABC):
```

Comment:
Are we using naming with s for plural elsewhere ?

Reply:
This is used to manage several examples at once, so I guess the plural is better